spec.html

```diff
@@ -406,16 +418,12 @@ <h1>Runtime Semantics: ClassElementEvaluation</h1>
       1. Let _formalParameterList_ be an instance of the production <emu-grammar>FormalParameters : [empty]</emu-grammar>.
       1. Let _closure_ be FunctionCreate(~Method~, _formalParameterList_, |FunctionBody|, _scope_, _strict_).
       1. Perform MakeMethod(_closure_, _homeObject_).
-      1. Perform SetFunctionName(_closure_, _key_, `"get"`).
+      1. Perform ! SetFunctionName(_closure_, _key_, `"get"`).
       1. <del>Let _desc_ be the PropertyDescriptor{[[Get]]: _closure_, [[Enumerable]]: _enumerable_, [[Configurable]]: *true*}.</del>
       1. <del>Return ? DefinePropertyOrThrow(_homeObject_, _propKey_, _desc_).</del>
+      1. <ins>Let _desc_ be ! DefaultMethodPropertyDescriptor(_key_, _closure_, _enumerable_).</ins>
```

Comment:
It's not valid to call DefaultMethodPropertyDescriptor here, since that abstract operation adds closure as `[[Value]]`, but we need to add it as `[[Get]]`.